service/controller/chart/resource/release/desired_test.go

```diff
@@ -224,7 +224,7 @@ func Test_DesiredState(t *testing.T) {
 			expectedState: ReleaseState{
 				Name:              "chart-operator-chart",
 				Status:            helmclient.StatusDeployed,
-				ValuesMD5Checksum: "8ccfa2ed7f5cb9a125b5f53254c296a8",
+				ValuesMD5Checksum: "2187a8fce91c3765a74d462062af7526",
```

Comment:
This means after we deploy this in CP, it would likely upgrade every charts, but this is needed.

Reply:
Thanks for the heads up but this is fine. We can verify once we've deployed that it only updates once.

Reply:
Since all CPs still have chart-operator with helm 2 support, I think it makes sense to test new branch based on `v0.13.x`. WDYT?

Reply:
Yes good point. Better to test with Helm 2 first.